### ISSUE TITLE
Bump cbindgen to 0.9.0

### ISSUE THREAD

**Issue by nbp** [mozilla/nixpkgs-mozilla#193]
Today, while trying to bump cbindgen to 0.9.0, I noticed the following issue:

```
these derivations will be built:
  /nix/store/47g9fcw1k66kzl29d4dvibq2jw4gd5ck-rust-cbindgen-0.9.0.drv
building '/nix/store/47g9fcw1k66kzl29d4dvibq2jw4gd5ck-rust-cbindgen-0.9.0.drv'...
unpacking sources
unpacking source archive /nix/store/bh97sc86a2x5yl8v66rfjk8g78bs53li-source
source root is source
unpacking source archive /nix/store/bm16vcmnpfw6yg697r2wz1kfjc58zxlp-rust-cbindgen-0.9.0-vendor
patching sources
configuring
[target."x86_64-unknown-linux-gnu"]
"linker" = "/nix/store/hpzj855nkgjvg58nrhq4910sb9q3kss1-gcc-wrapper-7.4.0/bin/cc"

building
++ env CC_x86_64-unknown-linux-gnu=/nix/store/hpzj855nkgjvg58nrhq4910sb9q3kss1-gcc-wrapper-7.4.0/bin/cc CXX_x86_64-unknown-linux-gnu=/nix/store/hpzj855nkgjvg58nrhq4910sb9q3kss1-gcc-wrapper-7.4.0/bin/c++ CC_x86_64-unknown-linux-gnu=/nix/store/hpzj855nkgjvg58nrhq4910sb9q3kss1-gcc-wrapper-7.4.0/bin/cc CXX_x86_64-unknown-linux-gnu=/nix/store/hpzj855nkgjvg58nrhq4910sb9q3kss1-gcc-wrapper-7.4.0/bin/c++ cargo build --release --target x86_64-unknown-linux-gnu --frozen
error: failed to acquire package cache lock

Caused by:
  failed to open: /homeless-shelter/.cargo/.package-cache

Caused by:
  Permission denied (os error 13)
```

This sounds similar to https://github.com/rust-lang/cargo/issues/6928

**Comment by nbp** [comment]
Taken from https://github.com/NixOS/nixpkgs/issues/61618 , this issue can be worked-around by adding:

```
  preConfigure = ''
    export HOME=`mktemp -d`
    mkdir -p $HOME/.cargo/
    touch $HOME/.cargo/.package-cache
  '';
```

**Comment by nbp** [comment]
This should be fixed by #194 .
